addons/auth_totp/wizard/auth_totp_wizard_views.xml

```diff
@@ -0,0 +1,76 @@
+            <form>
+                <sheet>
+                    <div class="o_auth_totp_enable_2FA container">
+                        <div class="mb-3 w-100">
+                            <h3 class="font-weight-bold">Authenticator App Setup</h3>
+                            <ul>
+                                <div class="d-md-none d-block">
+                                    <li>
+                                        <field class="text-wrap" name="url" widget="url" options="{'website_path': True}"
+                                        text="Click on this link to open your authenticator app"/></li>
+                                </div>
+                                <li>
+                                    <div class="d-flex align-items-center flex-wrap">
+                                        <span class="d-md-none d-block">Or install an authenticator app</span>
+                                        <span class="d-none d-md-block">Install an authenticator app on your mobile device</span>
+                                        <div class="d-block d-md-none">
+                                            <a href="https://play.google.com/store/search?q=authenticator&amp;c=apps" class="mx-2" target="blank">
+                                                <img alt="On Google Play" style="width: 24px;" src="/base_setup/static/src/img/logo_google_play.png"/>
+                                            </a>
+                                            <a href="http://appstore.com/2fa" class="mx-2" target="blank">
+                                                <img alt="On Apple Store" style="width: 24px;" src="/base_setup/static/src/img/logo_apple_store.png"/>
+                                            </a>
+                                        </div>
+                                    </div>
+                                </li>
+
+                                <span class="text-muted">Popular ones include Authy, Google Authenticator or the Microsoft Authenticator.</span>
+                                <li>Look for an "Add an account" button</li>
+                                <li>
+                                    <span class="d-none d-md-block">When requested to do so, scan the barcode below</span>
+                                    <span class="d-block d-md-none">When requested to do so, copy the key below</span>
+                                </li>
+                            </ul>
+
+                            <!-- Desktop version -->
+                            <div class="text-center d-none d-md-block">
+                                <field name="qrcode" readonly="True" widget="image"/>
+
+                                <h3 class="font-weight-bold"><a data-toggle="collapse"
+                                   href="#collapseTotpSecret" role="button" aria-expanded="false"
+                                   aria-controls="collapseTotpSecret">Cannot scan it?</a></h3>
```

Comment:
@dbeguin on master, if click on this link and then again to fold it -> JS crash?

```
Uncaught TypeError: scrollable is null
    scrolling.js:24
    scrollTo scrolling.js:24
    start scroller_service.js:46
    (Async: EventListener.handleEvent)
    addEventListener self-hosted:1175
    start scroller_service.js:22
    start env.js:106
    _startServices env.js:137
    startServices env.js:83
    AsyncFunctionNext self-hosted:692
    (Async: async)
    startWebClient start.js:32
    <anonymous> main.js:12
    def boot.js:289
    processJob boot.js:287
    processJobs boot.js:357
    def boot.js:299
    (Async: promise callback)
    def boot.js:296
    processJob boot.js:287
    processJobs boot.js:357
    def boot.js:299
    (Async: promise callback)
    def boot.js:296
    processJob boot.js:287
    processJobs boot.js:357
    define boot.js:162
    <anonymous> web.assets_backend_prod_only.js:90
```